src/reducers/rootReducer.js

```diff
@@ -2,7 +2,7 @@ const initState = {
     stories: [
         {usId: 'US301062', title: 'Add new Tower Ad Slot to SRP', Questions: 'Lorem ipsum, dolor sit amet consectetur adipisicing elit. Consequatur voluptate laborum perferendis, enim repellendus ipsam sunt autem at odit dolorum, voluptatum suscipit iste harum cum magni itaque animi laudantium fugiat'},
         {usId: 'US271107', title: 'Adobe: DFP Ad Tag for Custom Links', Questions: 'Lorem ipsum, dolor sit amet consectetur adipisicing elit. Consequatur voluptate laborum perferendis, enim repellendus ipsam sunt autem at odit dolorum, voluptatum suscipit iste harum cum magni itaque animi laudantium fugiat'},
-        {usId: 'US260509', title: 'AT Dependency (KBB Advertising) Used Car Landing Page - Add CPO content & CTA to certified inventory (to mitigate CPO landing and research pages going away)', Questions: ''}
+        {usId: 'US260509', title: 'AT Dependency (KBB Advertising) Used Car Landing Page - Add CPO content & CTA to certified inventory (to mitigate CPO landing and research pages going away)', Questions: 'Lorem ipsum, dolor sit amet consectetur adipisicing elit. Consequatur voluptate laborum perferendis, enim repellendus ipsam sunt autem at odit dolorum, voluptatum suscipit iste harum cum magni itaque animi laudantium fugiat'}
```

Comment:
I can't make a comment on the action file so that I add this here:
Please declare a constant for the static string `DELETE_STORY` so that you can re-use it in the reducer

Reply:
I declared a constant, but im not sure if thats the right way for react, please check my code.

Reply:
To use this constant here you have to export it from where you declared them import it here. 
Ex: 
```
export const DELETE_STORY = 'DELETE_STORY';
```
then 
```
import { DELETE_STORY } from 'path to source file'; 
```